### ISSUE TITLE
Import statement from Readme/Docs doesn't work 

### ISSUE THREAD

**Issue by stephenhmarsh** [sanity-io/sanity-plugin-hotspot-array#14]
Hello, thanks for the nice plugin.

I'm upgrading from Sanity Studio v2 to v3.

The [docs](https://www.sanity.io/plugins/hotspot-array) say:

```javascript
import { imageHotspotArray } from "sanity-plugin-hotspot-array";

export default defineConfig({
  // ...
  plugins: [
    imageHotspotArray(),
  ] 
})
```

That throws:
```
Uncaught error: Importing binding name 'imageHotspotArray' is not found.
:0:0
moduleDeclarationInstantiation@[native code]
link@[native code]
link@[native code]
linkAndEvaluateModule@[native code]
@[native code]
promiseReactionJob@[native code]
```

If I change the name to `imageHotspotArrayPlugin` it works/doesn't blow up, but I don't know for sure if that's the right export.

I'm using latest version of the plugin as recommended in the docs (not the v2 version).

**Comment by DanielHirunrusme** [comment]
Also receiving this error

**Comment by stephenhmarsh** [comment]
@DanielHirunrusme for v3 our working config ended up being:

```javascript
// sanity.config.js
import { imageHotspotArrayPlugin } from "sanity-plugin-hotspot-array";

..

export default defineConfig({
	...
	plugins: [
		imageHotspotArrayPlugin(),
	],
});

```

I think the names in the docs are just out of date/incorrect

**Comment by SimeonGriggs** [comment]
Fixed in #15
